test/fetch.test.js

```diff
@@ -1,39 +1,46 @@
-const { describe, it, beforeEach, before, after } = require('mocha')
-const { expect } = require('chai')
-const Runner = require('barnard59/lib/runner')
-const path = require('path')
-const clownface = require('clownface-io')
-const { namedNode } = require('@rdfjs/data-model')
-const toString = require('stream-to-string')
-const toStream = require('string-to-stream')
-const sinon = require('sinon')
-const proto = require('../lib/protoFetch')
+import { describe, it, beforeEach, before, after } from 'mocha'
+import chai from 'chai';
+import { create } from 'barnard59/lib/runner.js'
+import { resolve, dirname } from 'path'
+import clownface from 'clownface-io'
+import rdf from '@rdfjs/data-model';
+import toString from 'stream-to-string'
+import toStream from 'string-to-stream'
+import sinon from 'sinon';
+import proto, { fetch } from '../protoFetch.js'
```

Comment:
I don't think you can restructure a default export like that.

I'd rather export named and import like
```suggestion
import * as proto from '../protoFetch.js'
```

Reply:
Sure you can, [see](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#syntax): `import defaultExport, { export1 [ , [...] ] } from "module-name";`

Reply:
I'm pretty sure this only works when `export1` is a named export.

Reply:
Yep and `export1` is a named export.

Reply:
Oh, ok, `fetch` is exported both named and as `default { fetch }`. Why have it like that?